bin/ch/WScriptJsrt.cpp

```diff
@@ -1267,6 +1295,45 @@ JsErrorCode WScriptJsrt::FetchImportedModule(_In_ JsModuleRecord referencingModu
     return errorCode;
 }
 
+// Callback from chakracore to fetch module dynamically during runtime. In the test harness,
+// we are not doing any translation, just treat the specifier as fileName.
+// While this call will come back directly from runtime script or module code, the additional
+// task can be scheduled asynchronously that executed later.
+JsErrorCode WScriptJsrt::FetchImportedModuleFromScript(_In_ JsSourceContext dwReferencingSourceContext,
```

Comment:
updated in latest iteration

---
In reply to: [115327127](https://github.com/Microsoft/ChakraCore/pull/2913#discussion_r115327127) [](ancestors = 115327127)